src/app/component/user/components/profile/edit-profile/edit-profile.component.scss

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+
+.container {
+    margin-left: 135px;
+    max-width: 554px;
+    box-sizing: content-box;
+
+    h2 {
+        font-family: Montserrat, sans-serif;
```

Comment:
It is better to use scss variables for fonts

Reply:
fixed